DQM/SiStripMonitorDigi/src/SiStripMonitorDigi.cc

```diff
@@ -585,19 +607,21 @@ void SiStripMonitorDigi::analyze(const edm::Event& iEvent, const edm::EventSetup
     iSubdet->second.SubDetApvShots.clear();
   }
 
+  std::map<int,int> FEDID_v_digisum;
```

Comment:
why not just fill the histogram directly (and avoid the loop completely if its not needed?